content/en/blog/2022-08-26-manage-kyverno-policies-as-ocirepositories/index.md

```diff
@@ -0,0 +1,213 @@
+---
+author: developer-guy
+date: 2022-08-26 11:30:00+00:00
```

Comment:
Please set the date to today and we are ready to go :)

Reply:
Whee!